### PR TITLE
feat: propose new logo

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

**Review by cpaulve-1A — APPROVED** [review]

File: .attachments/otter.svg
Comment (cpaulve-1A) [inline .attachments/otter.svg:1]: super cute!

File: apps/vscode-extension/assets/logo-128x128.png
Comment (matthieu-crouzet) [inline apps/vscode-extension/assets/logo-128x128.png:1]: We should change the chrome extension logo as well

**Review by vscaiceanu-1a — DISMISSED** [review]
I love this beaver

File: apps/chrome-devtools/src/assets/extension/icons/16x16.png
Comment (fpaul-1A) [inline apps/chrome-devtools/src/assets/extension/icons/16x16.png:1]: should you the image be 16x16 ? (instead of 15x16)
same for the 32x32

**Review by fpaul-1A — DISMISSED** [review]

File: apps/chrome-devtools/src/assets/extension/icons/16x16.png
Comment (kpanot) [reply apps/chrome-devtools/src/assets/extension/icons/16x16.png:1]: Actually I have hesitated, I did not know where to add the empty column, but maybe on the left indeed

File: .attachments/logos/otter-fork-me.png
Comment (jbourgeois-1A) [inline .attachments/logos/otter-fork-me.png:1]: I expected a rickroll, I was disappointed

**Review by matthieu-crouzet — APPROVED** [review]

**Review by jbourgeois-1A — APPROVED** [review]

**Pull request merged**